vagrant/plugins/vagrant-hypconfigmgmt/lib/vagrant-hypconfigmgmt/command.rb

```diff
@@ -282,6 +284,13 @@ def ensure_vagrant_box_type_configured(env)
       update_settings(settings)
     end
 
+
+    def ensure_default_domain_configured(env)
+      settings = retrieve_settings()
+      settings['hostmanager']['default_domain'] ||= DEFAULT_DOMAIN
```

Comment:
that makes it so that local.yml files generated with older versions of vagrant-hypconfigmgmt will be compatible with this release. without it you'd get:
```
vdloo@workstation4:~/code/projects/hypernode-vagrant$ vagrant up 
/home/vdloo/code/projects/hypernode-vagrant/Vagrantfile:124:in `+': no implicit conversion of nil into String (TypeError)
	from /home/vdloo/code/projects/hypernode-vagrant/Vagrantfile:124:in `block (2 levels) in <top (required)>'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/v2/loader.rb:37:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/v2/loader.rb:37:in `load'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/loader.rb:113:in `block (2 levels) in load'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/loader.rb:107:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/loader.rb:107:in `block in load'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/loader.rb:104:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/config/loader.rb:104:in `load'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/vagrantfile.rb:153:in `machine_config'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/environment.rb:327:in `default_provider'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/plugin/v2/command.rb:174:in `block in with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/plugin/v2/command.rb:201:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/plugin/v2/command.rb:201:in `block in with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/plugin/v2/command.rb:183:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/plugin/v2/command.rb:183:in `with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/plugins/commands/up/command.rb:131:in `install_providers'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/plugins/commands/up/command.rb:85:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/cli.rb:42:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/lib/vagrant/environment.rb:302:in `cli'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.8.4/bin/vagrant:174:in `<main>'
```
until someone edits the local.yml and adds a default_domain